### PR TITLE
Gh-374: Add details of cache suffixes to changelog

### LINKED ISSUE #374: Add details of cache suffixes to changelog
Gaffer 2.0.0 introduced suffixes for graphs in a cache. This should be documented in the changelog.

### PR DESCRIPTION
**Pull request by GCHQDev404**

### CODE REVIEW

**Review by l46978 — COMMENTED** [review]
Would this not be better in the [Cache](https://github.com/gchq/gaffer-doc/blob/v2docs/docs/dev/components/cache.md)? We should probably review the branching strategy as this isn't technically part of the Gaffer 2.0.0 release.

**Comment by GCHQDeveloper314** [comment]
The cache suffix as a concept is part of the Gaffer 2.0 release.
The Cache page also needs more information to be migrated from the old docs. When that is done this can also be added to that page or moved to that page potentially and linked from the changelog.

**Comment by GCHQDeveloper314** [comment]
We may want to separate the part of this PR relating to features which are not in the Gaffer 2.0.0 release into a separate PR. This can be decided once the content of the 2.1.0 release is better understood - this may not be for a while.

File: docs/gaffer2.0/federation-changes.md [hunk 1]
```diff
@@ -238,3 +238,25 @@ By default, `GetSchema` results will be merged with `MergeSchema`. This returns
 | GetSchema         | MergeSchema                 |
 | GetTraits         | CollectionIntersect         |
 | others            | ConcatenateMergeFunction    |
+
+
+
+
+## SuffixCacheName 
+
+All Cache implementations are given a name and you can have multiple caches like NamedOperations, NamedViews, ect.
```
Comment (GCHQDeveloper314) [inline docs/gaffer2.0/federation-changes.md]:
```suggestion
All Cache implementations are given a name and you can have multiple caches like NamedOperations, NamedViews, etc.
```

File: docs/gaffer2.0/federation-changes.md [hunk 2]
```diff
@@ -238,3 +238,25 @@ By default, `GetSchema` results will be merged with `MergeSchema`. This returns
...
+
+However within FederatedStore where multiple graphs may exist, by default a cache name has the suffix of `graphId` added for identity and keeping caches for different graphs seperate e.g NamedOperationCache_graph1, NamedOperationCache_graph2, NamedOperationCache_graph3.
```
Comment (GCHQDeveloper314) [inline docs/gaffer2.0/federation-changes.md]:
```suggestion
However within FederatedStore where multiple graphs may exist, a cache name has the suffix of `graphId` added by default for identity and keeping caches for different graphs separate, e.g NamedOperationCache_graph1, NamedOperationCache_graph2, NamedOperationCache_graph3.
```

File: docs/gaffer2.0/federation-changes.md [hunk 3]
```diff
@@ -238,3 +238,25 @@ By default, `GetSchema` results will be merged with `MergeSchema`. This returns
...
+
+`gaffer.cache.service.default.suffix`
+
+The User can configure the individual cache suffix for a graph in the StoreProperties using the flags that takes priority over default.suffix.
+`gaffer.cache.service.named.operation.suffix`, 
+`gaffer.cache.service.job.tracker.suffix`, 
+`gaffer.cache.service.named.view.suffix`, 
+`gaffer.cache.service.federated.store.suffix`
+
+This also means two FederatedStores (or any graphs) can be made to share the same cache. In the example of load balancing FederatedStore instances.
```
Comment (GCHQDeveloper314) [inline docs/gaffer2.0/federation-changes.md]:
```suggestion
This also means two FederatedStores (or any graphs) can be made to share the same cache, e.g. load balancing FederatedStore instances.
```

**Review by GCHQDeveloper314 — APPROVED** [review]

**Pull request merged**